File: SchoolData/src/main/java/org/gb/homework/schooldata/view/SchoolDataView.java
package org.gb.homework.schooldata.view;

import org.gb.homework.schooldata.view.tools.ConsoleHelper;

//Меню
public class SchoolDataView {

    public int showMenu() {
        System.out.println("\nУчебное заведение");
        System.out.println("1. Учителя");
        System.out.println("2. Студенты");

        System.out.println("0. Выход");

        return ConsoleHelper.getNaturalInteger("Выберите номер задачи: ", x -> x >= 0 && x <= 2);
    }

    public void showWorkComplete() {
        System.out.println("Работа завершена...");
    }


    public void showTaskError() {
        System.out.println("Задача не выбрана");
    }
}

File: csharp/SchoolData/View/SchoolDataView.cs
using Gb.Homework.SchoolData.View.Tools;

namespace Gb.Homework.SchoolData.View
{
    public class SchoolDataView
    {

        public int ShowMenu()
        {

            Console.WriteLine("\nУчебное заведение");
            Console.WriteLine("1. Учителя");
            Console.WriteLine("2. Студенты");
            Console.WriteLine("3. Распределить студентов по учителям");

            Console.WriteLine("0. Выход");

            return ConsoleHelper.GetNaturalInteger("Выберите номер задачи: ", x => x >= 0 && x <= 3);
        }

        public void showWorkComplete()
        {
            Console.WriteLine("Работа завершена...");
        }


        public void showTaskError()
        {
            Console.WriteLine("Задача не выбрана");
        }
    }
}
